code/modules/food/food/snacks.dm

```diff
@@ -1390,7 +1408,8 @@
 
 /obj/item/weapon/reagent_containers/food/snacks/popcorn
 	name = "Popcorn"
-	desc = "Now let's find some cinema."
+	desc = "Now let's find a cinema."
```

Comment:
Nit: `some cinema` always implied a kind of ironic commentary on the drama of rounds to me, I think changing it loses something.